ExampleUsage/EnumExample.cs

```diff
@@ -23,6 +23,7 @@ public class EnumExample
 		public void Setup()
 		{
 			Database.SetInitializer(new DropCreateDatabaseAlways<MyDbContext>());
+			Database.SetInitializer(new DropCreateDatabaseAlways<MyCustomSchemaDbContext>());
```

Comment:
I wouldn't add the custom schema to the example usage as that's a more advanced feature, and less commonly used so might confuse people new to it. The example usage is intended to be a kind of quickstart.